src/deepsparse/pipeline_loggers/multi_logger.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A manager that oversees all the pipeline loggers
+"""
+
+from typing import Any, Dict, List, Union
+
+from deepsparse.pipeline_loggers.pipeline_logger import PipelineLogger
+from deepsparse.pipeline_loggers.prometheus_pipeline_logger import PrometheusLogger
+from deepsparse.timing.timing_schema import InferenceTimingSchema
+
+
+__all__ = ["MultiLogger"]
+
+LOGGERS = [PrometheusLogger]
+SUPPORTED_LOGGERS = {logger.identifier: logger for logger in LOGGERS}
+
+
+class MultiLogger(PipelineLogger):
+    """
+    Object that contains multiple loggers for
+    the given inference pipeline.
+
+    The envisioned lifecycle of a logger
+    manager:
```

Comment:
There are some naming problems that I will fix after the PR is approved.

Reply:
sounds good, not a fan of `MultiLogger` name